ui/src/app/settings/views/Images/WindowsForm/WindowsForm.test.js

```diff
@@ -47,7 +47,7 @@ describe("WindowsForm", () => {
     );
     expect(
       wrapper
-        .find("[name='windows_kms_host']")
+        .find("input[name='windows_kms_host']")
```

Comment:
Because these use `mount()` it was finding 3 items because there was a `FormikField[name='windows_kms_host']` which contained an `Input[name='windows_kms_host']` which contained an `input[name='windows_kms_host']`.